src/components/Epic.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import styled from 'preact-emotion';
+import pallete from 'pasteup/palette';
+import { headline } from 'pasteup/fonts';
+
+import SiteMessage from 'components/SiteMessage';
+
+const Content = styled('div')({
+    fontFamily: headline,
+    lineHeight: 1.3,
+    flexBasis: 630,
```

Comment:
Would it be better to define `flex-basis` in the `<CloseButton>`? Since the button has a hardcoded width, perhaps its `flex-basis` should be too?